AGENTS.md

```diff
@@ -0,0 +1,37 @@
+# BuildBuddy Coding Agent Guidelines
+
+## Build and Test
+- Use `./buildfix.sh` to format code
+- Use `bazel build` and `bazel test` to build and test the code
+- To run a specific test in a file with Bazel, use `bazel test --test_filter=<function_name/subtest_name> //specific/package:target`
+- Use `--test_output=errors` to reduce the Bazel command verbosity
+- Use `--config=remote-minimal` to run the builds and tests remotely. Only compatible with Linux-specific targets
```

Comment:
nit: the current wording suggests that only files named `*_linux_test.go` can run on `--config=remote-minimal`.

```suggestion
- Use `--config=remote-minimal` to run the builds and tests remotely (only compatible with targets that support Linux)
```